salt/modules/archive.py

```diff
@@ -936,7 +936,7 @@ def cmd_unzip(zip_file,
     if password:
         cmd.extend(['-P', password])
     if options:
-        cmd.append('{0}'.format(options))
+        cmd = cmd + options.split()
```

Comment:
I think we should we use `shlex.split` here.

Reply:
Yes, we need to use `shlex.split()`, and we should also be using `cmd.extend()` instead of `cmd = cmd + ....`, because the former will add to the existing list while the 2nd will unnecessarily allocate an extra list.